package.json

```diff
@@ -27,7 +27,6 @@
     "@asymmetrik/fhir-qb": "^0.12.5",
     "@asymmetrik/fhir-qb-mongo": "^0.12.3",
     "@projecttacoma/node-fhir-server-core": "^2.2.7",
-    "axios": "^0.21.4",
```

Comment:
This was a result of the connectathon so no fault here, but figured I'd mention. We actually do still need axios now since the ndjson workers are sending requests to get the ndjson content from the export. You'll see that when you rebase